src/app/DTT/email/index.js

```diff
@@ -1,17 +1,19 @@
-import { useState, useEffect } from 'react'
-
 import { Box, Input, Text, useClipboard, useToast } from '@chakra-ui/react'
-import { useMobileValue } from 'utils/hooks'
-import { getIsInvalid } from 'utils/functions'
+import { useEffect, useState } from 'react'
+
+import { ACTIONS } from 'utils/contexts/ErrorReducer'
 import Duplicates from './Duplicates'
+import { getIsInvalid } from 'utils/functions'
 import { useError } from 'utils/contexts/ErrorContext'
-import { ACTIONS } from 'utils/contexts/ErrorReducer'
+import { useMobileValue } from 'utils/hooks'
 
 const Write = ({ questionCode, data, onSendAnswer, regexPattern }) => {
+  // console.log(regexPattern)
```

Comment:
need to remove the hardcoded regex, probably my fault I left it there. But we need to remove it.